airflow/plugins_manager.py

```diff
@@ -137,8 +140,12 @@ def is_valid_plugin(plugin_obj, existing_plugins):
             # normalize root path as namespace
             namespace = '_'.join([re.sub(norm_pattern, '__', root), mod_name])
 
-            m = imp.load_source(namespace, filepath)
-            for obj in list(m.__dict__.values()):
+            spec = importlib.util.spec_from_file_location(namespace, filepath)
+            mod = importlib.util.module_from_spec(spec)
+            # get rid of mypy errors
+            assert isinstance(spec.loader, Loader)
```

Comment:
Since this is just to silence Mypy we should use https://mypy.readthedocs.io/en/stable/casts.html instead of an assertion

```suggestion
            cast(Loader, spec.loader)
```

Reply:
Fix it

Reply:
But CI failed sadly, https://travis-ci.org/apache/airflow/jobs/547026785